lib/ansible_tower_client/base_model.rb

```diff
@@ -157,6 +157,10 @@ def hashify(attribute)
       YAML.safe_load(send(attribute))
     end
 
+    def reload
```

Comment:
good catch @syncrou 

rails AR also replaces the current attributes. 👇 
would it be enough to replace `@raw_hash` ?

```
def reload(options = nil)
  self.class.connection.clear_query_cache

  fresh_object =
    if options && options[:lock]
      self.class.unscoped { self.class.lock(options[:lock]).find(id) }
    else
      self.class.unscoped { self.class.find(id) }
    end

  @attributes = fresh_object.instance_variable_get('@attributes')
  @new_record = false
  self
end
```